skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -732,6 +732,9 @@ espm::REFR::Data espm::REFR::GetData(
         result.teleport = (DoorTeleport*)data;
       else if (!memcmp(type, "XPRM", 4))
         result.boundsDiv2 = reinterpret_cast<const float*>(data);
+      else if (!memcmp(type, "XCNT", 4)) {
+        result.count = *(uint32_t*)data;
```

Comment:
`*reinterpret_cast<uint32_t*>(data)` should be a better solution for c++.

Reply:
Okay, I'll fix it when I'll return.